pkg/telemetry/sender_sio.go

```diff
@@ -0,0 +1,53 @@
+package telemetry
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/segmentio/analytics-go/v3"
+)
+
+const SegmentioEnvVariableName = "TESTKUBE_SEGMENTIO_KEY"
+
+var SegmentioKey = ""
```

Comment:
For Brew, it should contain default values, because ldflags are not used for providing it (by mantainer request)

Reply:
Good one! done.